src/App.tsx

```diff
@@ -1,7 +1,34 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends React.Component {
+  state = {
+    message: null,
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleOnKeyUp);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handleOnKeyUp);
+  }
+
+  handleOnKeyUp = (event: KeyboardEvent) => {
```

Comment:
To keep the code consistent with the task description, use 'event.code' instead of 'event.key'.